src/Admin/CitizenProjectCategoryAdmin.php

```diff
@@ -31,6 +37,32 @@ protected function configureFormFields(FormMapper $formMapper)
                     'Masqué' => BaseEventCategory::DISABLED,
                 ],
             ])
+            ->add('categorySkills', CollectionType::class, [
+                'label' => 'Liste des compétences',
+                'entry_type' => CategorySkillType::class,
+                'allow_add' => true,
+                'allow_delete' => true,
+                'by_reference' => false,
+                'error_bubbling' => false,
+                'attr' => [
+                    'class' => 'category-skills-bloc',
+                ],
+            ])
+        ;
+
+        $formMapper->getFormBuilder()
+            ->addEventListener(FormEvents::SUBMIT, function (FormEvent $event) {
+                $form = $event->getForm();
+                /** @var CitizenProjectCategory $citizenProjectCategory */
+                $citizenProjectCategory = $form->getData();
+
+                /** @var CitizenProjectCategorySkill $categorySkill */
```

Comment:
It's not needed if `getCategorySkills` has the right docblock like `CitizenProjectCategorySkill[]|Collection`